lib/crack.rb

```diff
@@ -1,18 +1,41 @@
 require_relative './enigma'
+require_relative './valid_input'
 
 ARGV == ['ecnrypted.txt', 'cracked.txt', 'date']
 ARGV[0] == 'encrypted.txt'
 ARGV[1] == 'cracked.txt'
 ARGV[2] == 'date'
 
-encrypted = File.open(ARGV[0], 'r')
-message = encrypted.read
-
+valid_input = ValidInput.new(ARGV[0], ARGV[1], ARGV[2])
 enigma = Enigma.new
-output = enigma.crack(message, ARGV[2])
 
-cracked = File.open(ARGV[1], 'w')
+encrypted_message = valid_input.check_valid_file
+
+if encrypted_message == 0
```

Comment:
provides a list of possible files if the one passed in as an argument does not exist in the current directory